ajax/libs/leaflet-groupedlayercontrol/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "filename" : "leaflet.groupedlayercontrol.min.js",
+  "version": "0.6.0",
+  "description": "A Leaflet layer control with support for grouping overlays together",
+  "main": "dist/leaflet.groupedlayercontrol.min.js",
+  "directories": {
+    "example": "example"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "//git:github.com/ismyrnow/leaflet-groupedlayercontrol",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "leaflet.groupedlayercontrol.min.css"
+          "leaflet.groupedlayercontrol.min.js"
+          "leaflet.groupedlayercontrol.min.js.map"
+        ]
+      }
+    ]
+  },
+  "scripts": {
+    "test": "eslint src",
+    "build": "grunt"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ismyrnow/leaflet-groupedlayercontrol.git"
+  },
+  "keywords": [
+    "leaflet",
+    "overlay",
+    "groups",
+    "grouped",
+    "control",
+    "plugin"
+  ],
+  "author": "Ishmael Smyrnow",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ismyrnow/leaflet-groupedlayercontrol/issues"
+  },
+  "homepage": "https://github.com/ismyrnow/leaflet-groupedlayercontrol",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.